From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 30 May 2011 20:21:12 +0100
Subject: hardwire plugin directory

Set directory to scan for plugins subdirectory containing plugins to
system directory in which the plugins subdirectory is installed.
---
 MLDemos/mldemos.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MLDemos/mldemos.cpp b/MLDemos/mldemos.cpp
index 974148a..003bd06 100644
--- a/MLDemos/mldemos.cpp
+++ b/MLDemos/mldemos.cpp
@@ -597,7 +597,7 @@ void MLDemos::initDialogs()
 void MLDemos::initPlugins()
 {
     qDebug() << "Importing plugins";
-    QDir pluginsDir = QDir(qApp->applicationDirPath());
+    QDir pluginsDir = QDir("/usr/lib/mldemos");
     QStringList pluginFileNames;
     QDir alternativeDir = pluginsDir;
 
